source/common/config/type_to_endpoint.cc

```diff
@@ -94,6 +92,7 @@ const Protobuf::MethodDescriptor& sotwGrpcMethod(absl::string_view type_url) {
 
 const Protobuf::MethodDescriptor& restMethod(absl::string_view type_url) {
   const auto it = typeUrlToServiceMap().find(static_cast<std::string>(type_url));
+  ENVOY_LOG_MISC(debug, "HTD {}", type_url);
```

Comment:
What does HTD mean?

Reply:
HT debug :) I've removed the occurrences in the tree.